.github/ISSUE_TEMPLATE/bug-template.yaml

```diff
@@ -17,6 +17,7 @@ body:
       label: Which Cloudflare product(s) does this pertain to?
       multiple: true
       options:
+        - C3
```

Comment:
can you add the same to the feature request template as well? thanks!

Reply:
The feature request template does not have a drop down.

Reply:
<img width="895" alt="image" src="https://github.com/cloudflare/workers-sdk/assets/15655/75046aa4-5cf4-4d12-b292-c1010f1c8668">

Reply:
ah. bad assumption on my part. my apologies.